src/components/app/MenuButtons/index.css

```diff
@@ -35,3 +35,9 @@
 .menuButtonsLink:hover {
   background-color: rgba(0, 0, 0, 0.1);
 }
+.open-in-new{
+  background-image: url(../../../../res/img/svg/open-in-new.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 10px;
```

Comment:
can you just add a background-size, like this: `background-size: 11px` ?
I use `11px` because this is the font-size here, and when I tried it it looked good.